src/windows/multiplayer.c

```diff
@@ -264,7 +264,7 @@ static void window_multiplayer_groups_show_group_dropdown(rct_window *w, rct_wid
 
 	for (i = 0; i < network_get_num_groups(); i++) {
 		gDropdownItemsFormat[i] = 2777;
-		gDropdownItemsArgs[i] = (sint64)network_get_group_name(i);
+		gDropdownItemsArgs[i] = (int)network_get_group_name(i);
```

Comment:
```C
/home/janisozaur/workspace/OpenRCT2/src/windows/multiplayer.c: In function ‘window_multiplayer_groups_show_group_dropdown’:
/home/janisozaur/workspace/OpenRCT2/src/windows/multiplayer.c:267:27: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
   gDropdownItemsArgs[i] = (sint64)network_get_group_name(i);
                           ^
```

I'm a little uneasy about this change, but I've decided to do it anyway, as we do have it done elsewhere and x86 is our primary target for now.